src/main/java/org/jivesoftware/smackx/pubsub/PubSubIntegrationTest.java

```diff
@@ -915,4 +923,44 @@ public void transientNotificationOnlyNodeWithItemTest() throws NoResponseExcepti
             pubSubManagerOne.deleteNode(nodename);
         }
     }
+
+    @SmackIntegrationTest
+    public void isEqualToOneTest() {
+        assertEquals(1,1);
+    }
+
+    @SmackIntegrationTest
```

Comment:
Please add javadoc that clearly defines what this method intends to test. Without such a description, it is hard for other developers to maintain this code!